binlog_streamer.go

```diff
@@ -230,6 +231,22 @@ func (s *BinlogStreamer) Run() {
 			// the full query that was executed on the master (with annotations)
 			// that is otherwise not possible to reconstruct
 			query = ev.Event.(*replication.RowsQueryEvent).Query
+		case *replication.QueryEvent:
+			query = ev.Event.(*replication.QueryEvent).Query
+			if string(query) == "BEGIN" {
```

Comment:
Can you leave a comment on why this is ignored here? It's not super obvious to me.